packages/next/src/server/base-server.ts

```diff
@@ -596,66 +564,54 @@ export default abstract class Server<
 
     this.setAssetPrefix(assetPrefix)
     this.responseCache = this.getResponseCache({ dev })
-  }
-
-  protected reloadMatchers() {
-    return this.matchers.reload()
-  }
 
-  private handleRSCRequest: RouteHandler<ServerRequest, ServerResponse> = (
-    req,
-    _res,
-    parsedUrl
-  ) => {
-    if (!parsedUrl.pathname) return false
-
-    if (this.normalizers.prefetchRSC?.match(parsedUrl.pathname)) {
-      parsedUrl.pathname = this.normalizers.prefetchRSC.normalize(
-        parsedUrl.pathname,
-        true
+    // Setup the request adapter.
+    if (
+      this.minimalMode &&
+      process.env.NEXT_RUNTIME !== 'edge' &&
+      this.nextConfig.output === 'standalone'
+    ) {
+      this.requestAdapter = new StandaloneRequestAdapter(
+        new XMatchedPathRequestAdapter(
```

Comment:
nit: I feel like dropping `X` for both of these adapters and just calling them `MatchedPathRequestAdapter` and `InvokePathRequestAdapter` is nicer to read.